compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEDistributeL1Allocations.cpp

```diff
@@ -25,60 +26,38 @@ using namespace mlir;
 
 namespace {
 
-/// Find all induction variables of all `scf.forall` ops that are mapped to
-/// gpu thread dimensions (as opposed to gpu block dimensions etc).
-FailureOr<DenseSet<Value>> getThreadIndVars(ModuleOp moduleOp) {
-  DenseSet<Value> threadIndVars;
+/// Find all `scf.forall` ops that are mapped to gpu thread dimensions (as
+/// opposed to gpu block dimensions etc) i.e. the ones which will be mapped to
+/// cores.
+DenseSet<Operation *> getCoreForallOps(ModuleOp moduleOp) {
```

Comment:
I think you can return `DenseSet<scf::ForallOp>` instead

Reply:
Done.